scripts/gtcadis.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
+import argparse
+import yaml
```

Comment:
I suggest `ruamel.yaml`, docs here http://yaml.readthedocs.io/en/latest/

Reply:
For the current use case, is there motivation to add an external dependency (given that not everyone lives in a conda world)?  This `yaml` appears to be relatively standard (I guess I don't really know)

Reply:
either one is fine, just my recommendation!